src/Bridge/Doctrine/Orm/Filter/BooleanFilter.php

```diff
@@ -91,7 +108,7 @@ public function getDescription(string $resourceClass) : array
             $properties = array_fill_keys($this->getClassMetadata($resourceClass)->getFieldNames(), null);
         }
 
-        foreach ($properties as $property => $boolean) {
+        foreach ($properties as $property => $unused) {
```

Comment:
I think so, on the other hand it's slower (one more function call). Keep it as is.